spring-boot-tools/spring-boot-loader-tools/src/main/java/org/springframework/boot/loader/tools/layout/BaseLayout.java

```diff
@@ -0,0 +1,29 @@
+package org.springframework.boot.loader.tools.layout;
```

Comment:
Dedicating new package for layouts because referencing nested classes in Gradle is tricky (you need to use dollar which must be escaped in Gradle, and also, this is difference between Maven and Gradle, hence confusing...)